### DISCUSSION TITLE
To define wich folder are to use on OneDrive for BackUp

### DISCUSSION THREAD

**Discussion by city12001** [lavinir/hassio-onedrive-backup#214] [Ideas]
I have the issue, that the standard folder in OneDrive isn´t the root, i don´t know why.
For me it is a greate functuion, if i can define the destination folder in OneDrive.
Thanks for your help in Advance

**Comment by Maschler** [comment]
First i´d like to thank you for your great work.

Because i have a special folder structure in onedrive, for me it would be also great having the opportunity to change the folder where to save the Backup.

**Comment by lavinir** [comment]
Yeah this has been discussed multiple times in the past. The reason for this is permissions. The way the Special App Folder works is that each App gets their own special folder under the Apps Folder that they have full RW access to and that's it. 
That is what allows me to only request that permission on our OneDrive account. The alternative would be to request access to your Entire OneDrive which I will not do as I wouldn't want to give those permissions as a user myself.

**Comment by GazTheGeek** [comment]
Ditto from me. I have 2 instances of HA so being able to specify the onedrive folder for each instance would stop the addon including both instances in the backup count.

**Reply by lavinir** [reply]
Actually this is supported. 
see [here](https://github.com/lavinir/hassio-onedrive-backup?tab=readme-ov-file#backup-instance-name-optional)

**Comment by AxReds** [comment]
Same here. I have a special OneDrive folder dedicated to any backup I do on my network and I'd like to have Hassio OneDrive Backup to comply to this rule too.
